test/get_previous_releases.py

```diff
@@ -122,6 +122,9 @@ def progress_hook(progress_bytes, total_size):
         if response.status != 200:
             raise RuntimeError(f"HTTP request failed with status code: {response.status}")
 
+        sock_info = response.fp.raw._sock.getpeername()
+        print(f"Connected to {sock_info[0]}")
```

Comment:
I'm fine with this as well, alternatively to document what we're connecting to, we could unpack the `ip` from the `(host, port)`/`(host, port, flowinfo, scope_id)` tuples with something like:
```suggestion
        ip, *_ = response.fp.raw._sock.getpeername()
        print(f"Connected to {ip}")
```
or
```suggestion
        ip = response.fp.raw._sock.getpeername()[0]
        print(f"Connected to {ip}")
```